src/plugins/intel_cpu/src/nodes/transpose.cpp

```diff
@@ -154,6 +227,10 @@ void MKLDNNTransposeNode::createPrimitive() {
         IE_THROW() << "Preferable primitive descriptor was not set.";
 
     if (getParentEdgeAt(INPUT_DATA_IDX)->getMemory().getDesc().hasLayoutType(LayoutType::ncsp) &&
+        getChildEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::ncsp) &&
+        order == std::vector<size_t>{0, 3, 1, 2}) {
+        performAsReorder = true;
```

Comment:
minor
maybe we need execute 240 - 246 lines only if `performAsReorder  == false`